assets/trix/stylesheets/toolbar.scss

```diff
@@ -15,6 +15,7 @@ trix-toolbar {
     display: flex;
     flex-wrap: nowrap;
     justify-content: space-between;
+    overflow: auto;
```

Comment:
I'm 👍 on allowing overflow. Do you think using `overflow-x` would be more intention revealing?

Reply:
Absolutely! I will happily update.

Reply:
Change from `overflow: auto` to `overflow-x: auto` squashed and pushed.